pip.ado

```diff
@@ -420,19 +414,13 @@ qui {
 	
 	
 	if ("`aggregate'" != "") {
-		if ("`ppp_year'" != ""){
-			noi di  as err "Option PPP cannot be combined with aggregate."
-			error 198
-		}
 		noi disp as res "Note: " as text "Aggregation is only possible over reference years."
 		local agg_display = "Aggregation in base year(s) `year'"
 	}
 	
-	if ("`ppp_year'" != "") {
-		if (wordcount("`country'")>2) {
-			noi di as err "Option PPP can only be used with one country."
-			error 198
-		}
+	if (wordcount("`country'") > 2) {
+		noi di as err "Option PPP can only be used with one country."
+		error 198
```

Comment:
This condition does not make any sense. We should not limit the number of countries.